solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -30,6 +30,18 @@ interface ILiquidityPoolV1Converter is IConverter {
         uint256[] memory _reserveMinReturnAmounts
     ) external;
 
+    function addLiquidity(
+        IERC20Token[2] memory _reserveTokens,
+        uint256[2] memory _reserveAmounts,
+        uint256 _minReturn
+    ) external payable;
```

Comment:
It's important that both public `addLiquidity` and `removeLiquidity` methods returns positions ids for the LM implementation

Reply:
And is it important to do this prior the LM implementation, in order to avoid breaking the API?
In this case, is LM designated for SP converters only? Because we already have many other converters deployed, so if this is not the case, then we'll be breaking the API anyway.

Reply:
Ah wait, this is the LP contract; ignore...